core/src/main/java/com/alibaba/alink/operator/stream/onlinelearning/FtrlPredictStreamOp.java

```diff
@@ -72,15 +72,7 @@ public void flatMap(Row row, Collector<Tuple2<Integer, Row>> out)
                             out.collect(Tuple2.of(i, row));
                         }
                     }
-                }).partitionCustom(new Partitioner<Integer>() {
```

Comment:
In alink, we follow a common practice that use explicit functional object instead of lambda. This is mainly about issues of serialization.

Reply:
I see, thank you